src/org/zaproxy/zap/extension/todo/resources/Messages.properties

```diff
@@ -0,0 +1,8 @@
+# An example ZAP extension which adds a right click menu item to all of the main
```

Comment:
This example content can be removed.

Those items earlier noted as needing i18n treatment can (and should) be added here.

Eg: `removeBtn.setText("Remove");` should become: `removeBtn.setText(Constant.messages.getString("ext.todo.remove.btn.label"));` and add: `ext.todo.remove.btn.label=Remove` here, etc.